### ISSUE TITLE
Scale data --> Normalize and denormalize

### ISSUE THREAD

**Issue by maytepenella** [jsantalo/happybirds#28]
"It is usually a good idea to scale the data for SVM training"

We should look into normalization/denormalization for the features, because we could have features that appear more than once and other that will only appear once, for example if we consider emojis or puntuation signs versus words or bigrams.

We can use "StandardScaler" from **sklearn.preprocessing**

From documentation of "StandardScaler":

* Centering and scaling happen independently on each feature by computing the relevant statistics on the samples in the training set. Mean and standard deviation are then stored to be used on later data using the transform method.

* Standardization of a dataset is a common requirement for many machine learning estimators: they might behave badly if the individual feature do not more or less look like standard normally distributed data (e.g. Gaussian with 0 mean and unit variance).

**Comment by maytepenella** [comment]
Only done normalization. I think demormalization is not needed as this is a classification problem. 

With this normalization, only increased 1 % (it could be no improvement as I only did 1 iteration)

**Comment by maytepenella** [comment]
added functions "normalize_train_data" and "normalize_validate_data" in trans.py
